docs-2.0/3.ngql-guide/13.edge-statements/1.insert-edge.md

```diff
@@ -1,5 +1,11 @@
 # INSERT EDGE
 
+The `INSERT EDGE` statement inserts an edge from a source vertex (given by src_vid) to a destination vertex (given by dst_vid).
```

Comment:
The `INSERT EDGE` statement inserts an edge from a source vertex (given by src_vid) to a destination vertex (given by dst_vid) with a given rank.